packages/common-settings-bridge/README.md

```
@@ -0,0 +1,1146 @@
+  user: "twake"
+  password: "twake_password"
+```
+
+SQLite (for development):
+```yaml
+database:
+  engine: "sqlite"
+  path: "/data/bridge.db"
+```
+
+**RabbitMQ Configuration**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)